### PR TITLE
feat: move to turborepo, init docs

### PR DESCRIPTION
**Pull request by alisajadih**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -2,5 +2,5 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 yarn lint-staged && \
```
Comment (mdeljavan) [inline .husky/pre-commit:4]:
```suggestion
yarn lint-staged
```

File: .husky/pre-commit [hunk 2]
```diff
@@ -2,5 +2,5 @@
...
-yarn nx affected --target=type-check && \
-yarn nx affected --target=lint
+yarn type-check&& \
```
Comment (mdeljavan) [inline .husky/pre-commit:5]:
```suggestion
yarn type-check
```

File: apps/docs/package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "docs",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "yarn run -T vite",
+    "build": "yarn run -T vite build",
+    "test": "yarn run -T vitest --run",
+    "lint": "eslint --max-warnings=0 \"**/*.{js,jsx,ts,tsx}\"",
```
Comment (mdeljavan) [inline apps/docs/package.json:9]:
```suggestion
    "lint": "yarn -T eslint --max-warnings=0 \"**/*.{js,jsx,ts,tsx}\"",
```

File: package.json [hunk 4]
```diff
@@ -6,19 +6,21 @@
...
-    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
-    "format:check": "prettier --check \"**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
-    "lint": "eslint --max-warnings=0 \"**/*.{js,jsx,ts,tsx}\"",
-    "start": "nx serve",
-    "build": "nx build",
-    "test": "nx test",
     "storybook": "start-storybook -p 4400",
-    "build-storybook": "build-storybook",
-    "chromatic": "chromatic"
+    "chromatic": "chromatic",
```
Comment (mdeljavan) [inline package.json:15]: uninstall chromatic

File: package.json [hunk 5]
```diff
@@ -31,15 +33,19 @@
     "@changesets/cli": "^2.23.2",
     "@commitlint/cli": "^17.0.3",
     "@commitlint/config-conventional": "^17.0.3",
-    "@front.zen/design-system": "*",
+    "@front.zen/design-system": "^0.0.1",
```
Comment (mdeljavan) [inline package.json:36]: move to .storybook

File: libs/core/package.json [hunk 6]
```diff
@@ -15,5 +24,6 @@
     ".": {
       "default": "./src"
```
Comment (mdeljavan) [inline libs/core/package.json:25]:
```suggestion
      "default": "./dist"
```

File: libs/core/package.json [hunk 7]
```diff
@@ -15,5 +24,6 @@
...
     }
-  }
+  },
+  "types": "./src/index.ts"
```
Comment (mdeljavan) [inline libs/core/package.json:28]: check it

File: .storybook/tsconfig.json [hunk 8]
```diff
@@ -1,14 +1,4 @@
...
-    "../**/*.test.js",
-    "../**/*.spec.ts",
-    "../**/*.test.ts",
-    "../**/*.spec.tsx",
-    "../**/*.test.tsx",
-    "../**/*.spec.jsx",
-    "../**/*.test.jsx"
-  ],
-  "include": ["../**/*"]
+  "include": ["./*"]
```
[the remark this answers is not in the document]
Comment (alisajadih) [reply .storybook/tsconfig.json:3]: it does not work

File: turbo.json [hunk 9]
```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": ["dist/**"]
```
[the remark this answers is not in the document]
Comment (alisajadih) [reply turbo.json:6]: ack

**Review by mdeljavan — APPROVED** [review]

File: .prettierignore [hunk 10]
```diff
@@ -1,5 +1,53 @@
-# Add files here to ignore them from prettier formatting
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```
[the remark this answers is not in the document]
Comment (alisajadih) [reply .prettierignore:1]: i made a softlink

**Review by mdeljavan — APPROVED** [review]

**Review by mdeljavan — CHANGES_REQUESTED** [review]
why did you remove the .prettierignore file?

**Review by mdeljavan — APPROVED** [review]

**Pull request merged**